test/api/buffer/api_test.go

```diff
@@ -105,8 +105,107 @@ func RunFunctionalTest(t *testing.T, testDir, workingDir string, binary string)
 	// Replace all %%ENV_VAR%% in all files in the working directory
 	testhelper.MustReplaceEnvsDir(workingDirFs, `/`, envProvider)
 
+	// Run API server
+	apiUrl, apiEnvs, stdout, stderr := RunApiServer(t, binary, project.StorageAPIHost())
+
+	// Connect to the etcd
+	etcdClient := etcdhelper.ClientForTestFrom(
+		t,
+		apiEnvs.Get("BUFFER_ETCD_ENDPOINT"),
+		apiEnvs.Get("BUFFER_ETCD_USERNAME"),
```

Comment:
Presunul som kod, ktory nemal velmi co robit v `RunRequests` do nadradeneho `RunE2ETest` (v tomto commite v Buffer API, v dalsom v Templates API).

Zatial je to duplicitne, je task na zjednotenie E2E API testov v cooldown cykle.